client/src/Components/Registration.js

```diff
@@ -50,69 +70,139 @@ const Registration = () => {
 
   const handleSubmit = e => {
     e.preventDefault();
-    dispatch(register(email, password, confirmPassword, handle)).then(res => {
+    dispatch(register(email, password, confirmPassword, name)).then(res => {
       console.log("res", res);
```

Comment:
Still need this?

Reply:
I tested it and there's an error in the backend with postman when I try to get rid of it.
I'll leave it in for now; it could be developed upon further in a new branch .